### PR TITLE
docs: Update Readme document

### LINKED ISSUE #3791: update readme document

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the README document by using the proper link for
the contributing guide as well as a misspelling.

Fixes #3791

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by cmaf — CHANGES_REQUESTED** [review]
@GabyCT minor comment on usage of "how to" vs "how-to"

File: docs/README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ See the [tracing documentation](tracing.md).
 * [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
 [`runc`](https://github.com/opencontainers/runc).
 
-### Howto guides
+### How to guides
```
Comment (cmaf) [inline docs/README.md:24]: @GabyCT I think it should be "How-to" in this case since it is not a verb

File: docs/README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ See the [tracing documentation](tracing.md).
...
 
-See the [howto documentation](how-to).
+See the [how to documentation](how-to).
```
Comment (cmaf) [inline docs/README.md:26]: Also "How-to" here

**Review by cmaf — APPROVED** [review]
@GabyCT lgtm

**Review by devimc — APPROVED** [review]

**Pull request merged**